client/packages/system/src/Vaccination/api/useVaccination.ts

```diff
@@ -43,34 +45,13 @@ export function useVaccination({
   vaccinationId: string | undefined;
   defaultClinician?: Clinician;
 }) {
-  const { store, storeId, api } = useVaccinationsGraphQL();
+  const { store } = useVaccinationsGraphQL();
 
-  const { data: dose, isLoading: doseLoading } = useQuery({
-    queryKey: [VACCINATION, vaccineCourseDoseId],
-    queryFn: async () => {
-      const result = await api.vaccineCourseDose({
-        doseId: vaccineCourseDoseId,
-      });
-
-      if (result.vaccineCourseDose.__typename === 'VaccineCourseDoseNode') {
-        return result.vaccineCourseDose;
-      }
-    },
-  });
-  const { data: vaccination, isLoading: vaccinationLoading } = useQuery({
-    queryKey: [VACCINATION, vaccinationId],
-    queryFn: async () => {
-      if (!vaccinationId) {
-        return null;
-      }
-      const result = await api.vaccination({ vaccinationId, storeId });
+  const { data: dose, isLoading: doseLoading } =
+    useDoseQuery(vaccineCourseDoseId);
```

Comment:
heck sorry more rearranging, moves the guts of these queries outside the top level hook, was getting unwieldy